source/developer/fx-guidelines.rst

```diff
@@ -29,13 +29,13 @@ Fast has two parts: being "responsive" and being "quick".
 
 **Obvious**
 
-There is no such thing as "user error"--it’s always our fault as product designers. We stop problems from incorrect usage by making users aware of correct usage. Fixes could be through user interface improvements, adjustments to help text, or helping users recover when things go wrong–adding undelete, undo and rolling back for example.
+There is no such thing as "user error"--it’s always our fault as product designers. We stop problems from incorrect usage by making users aware of correct usage. Fixes could be through user interface improvements, adjustments to help text or helping users recover when things go wrong–for example, adding undelete, undo and rolling back.
```

Comment:
For better readability should we change the second sentence to read: "We prevent problems of incorrect usage by..."?

Reply:
Thank you Lindy! I've made all the changes (this and the ones below).